apps/aecore/lib/aecore/governance/governance_constants.ex

```diff
@@ -46,7 +44,7 @@ defmodule Aecore.Governance.GovernanceConstants do
 
   @default_tx_gas_price 15_000
 
-  @allowed_txs_types [
+  @known_tx_types [
```

Comment:
why is this "governance" constant? I think of this more related to the transactions calculation.

Reply:
"known..." doesn't explain well what its used for and why we would need it

Reply:
Could you suggest a proper name then?

Reply:
right now its only used for gas calculation, so maybe `gas_calculation_tx_types`

Reply:
Thank you for your clarifications. Well, when I was defining this constant, I also thought about that it could be reused in somewhere else too , that's why I used more generic name for it. I will change it.